src/charactersheet/viewmodels/character/wealth/view.html

```diff
@@ -111,6 +112,43 @@
           <b data-bind="text: $component.quickSpendMessage"></b>
         </form>
       </div>
+      <div class="col-xs-12">
+        <form data-bind="submit: $component.quickEarn">
+          <div class="input-group input-group-sm d-flex">
+            <input
+              type="number"
+              minvalue="1"
+              required
+              class="form-control d-flex-4"
+              placeholder="Quick earn"
+              data-bind="value: $component.quickEarnAmount"
+            >
+            <span class="input-group-btn d-flex-3">
+              <select
+                required
+                class="form-control"
+                data-bind="value: $component.quickEarnCoinType"
+              >
+                <option value="copper">CP</option>
+                <option value="silver">SP</option>
+                <option value="electrum">EP</option>
+                <option value="gold" selected>GP</option>
+                <option value="platinum">PP</option>
+              </select>
+            </span>
+            <span class="input-group-btn d-flex-3">
+              <button
+                style="width: 100%"
+                class="btn btn-default"
+                type="submit"
+              >
+                Earn
```

Comment:
This UI works, but I always imagined the quick spend/earn like the HP module.
[ - ] [ Amount ] [ Denom ] [ + ]
Is that too farfetched? or just a bad idea?

Reply:
We tried something like that but the dropdown for GP/PP/etc makes it really unwieldy and ugly. Dave actually did both and we agreed this was more clear.

Reply:
sounds good. Just wanted to make sure it was considered.